safe_cli/operators/safe_operator.py

```diff
@@ -228,6 +232,12 @@ def __init__(self, address: ChecksumAddress, node_url: str):
         self.require_all_signatures = (
             True  # Require all signatures to be present to send a tx
         )
+        try:
```

Comment:
Could you put this logic into a function? (we should do the same for tx service and so on)

Like in SafeOperator:
  def load_ledger_manager() -> Optional[LedgerManager]

So __init__ is more clean

Reply:
Yes, but I cannot define the return value without import it